commons/src/main/scala/com/lynbrookrobotics/potassium/commons/flywheel/DoubleFlywheel.scala

```diff
@@ -0,0 +1,91 @@
+package com.lynbrookrobotics.potassium.commons.flywheel
+
+import com.lynbrookrobotics.potassium.{Component, PeriodicSignal, Signal, SignalLike}
+import com.lynbrookrobotics.potassium.control.{PIDF, PIDFConfig}
+import com.lynbrookrobotics.potassium.tasks.{ContinuousTask, FiniteTask}
+import com.lynbrookrobotics.potassium.units.{GenericDerivative, GenericValue}
+
+import squants.{Angle, Dimensionless}
+import squants.motion.AngularVelocity
+
+trait DoubleFlywheelProperties {
+  def velocityGains: PIDFConfig[AngularVelocity,
+                                GenericValue[AngularVelocity],
+                                AngularVelocity,
+                                GenericDerivative[AngularVelocity],
+                                Angle,
+                                Dimensionless]
+}
+
+trait DoubleFlywheelHardware {
+  def leftVelocity: Signal[AngularVelocity]
+  def rightVelocity: Signal[AngularVelocity]
+}
+
+abstract class DoubleFlywheel {
+  type Properties <: DoubleFlywheelProperties
```

Comment:
We want to allow the user to be able to specify their specific subtype of `Properties` for additional type safety in case there are multiple configurations for different flywheels.